From 85821aab39b3403a8b5731812a930b78684d1642 Mon Sep 17 00:00:00 2001
From: Hugh Dickins <hughd@google.com>
Date: Mon, 25 Jul 2011 17:12:23 -0700
Subject: [PATCH] mm: truncate functions are in truncate.c
Git-commit: 85821aab39b3403a8b5731812a930b78684d1642
Patch-mainline: 3.1-rc1
References: bnc#731999

Correct comment on truncate_inode_pages*() in linux/mm.h; and remove
declaration of page_unuse(), it didn't exist even in 2.2.26 or 2.4.0!

Signed-off-by: Hugh Dickins <hughd@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Acked-by: Michal Hocko <mhocko@suse.cz>

---
 include/linux/mm.h |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

Index: linux-3.0-bnc731999-shm-rewrite/include/linux/mm.h
===================================================================
--- linux-3.0-bnc731999-shm-rewrite.orig/include/linux/mm.h
+++ linux-3.0-bnc731999-shm-rewrite/include/linux/mm.h
@@ -1455,8 +1455,7 @@ extern int do_munmap(struct mm_struct *,
 
 extern unsigned long do_brk(unsigned long, unsigned long);
 
-/* filemap.c */
-extern unsigned long page_unuse(struct page *);
+/* truncate.c */
 extern void truncate_inode_pages(struct address_space *, loff_t);
 extern void truncate_inode_pages_range(struct address_space *,
 				       loff_t lstart, loff_t lend);
